core/src/crds_gossip_pull.rs

```diff
@@ -26,6 +26,60 @@ use std::collections::HashMap;
 use std::collections::VecDeque;
 
 pub const CRDS_GOSSIP_PULL_CRDS_TIMEOUT_MS: u64 = 15000;
+// determines how many pull request filters are generated per pull request.
+pub const CRDS_GOSSIP_PULL_SPLIT_COUNT: u64 = 4;
```

Comment:
@aeyakovenko maybe I should make this number scale. I can figure out how many items fit in a bloom of a given size. If I have more values than that, I can bump up the number of filters I generate? Currently I have 4 different filters all the time, can technically fit 4k different crds values.

Reply:
This number seems insufficient as the size of the mask grows large. In expectation you will need `.57(2^mask_bits)` number of queries to cover the search space. Do we want some fallback plan when this number gets large?

Reply:
Previously with 64K blobs we could fit around ~50k Hashes into the bloom. 50k Hashes in terms of 1K CrdsFilters results in 6 mask_bits. 

Which according to `0.57(2^6)` is 7. I think if I limit it to 8 random filters; i.e 8 pull requests we should maintain 64k blob level fidelity. wdyt?

Reply:
Previously with 64K blobs we could fit around ~50k Hashes into the bloom. 50k Hashes in terms of 1K CrdsFilters results in 7 mask_bits. 

Which according to `0.57(2^7)` is 73 filters. That sounds like a lot of pull requests. what do you guys think ?  Puts a lot of load on the server since they will have to test every crdsvalue against each filter in each pull.